cupy_backends/cuda/libs/cudnn.pyx

```diff
@@ -6,6 +6,7 @@ cimport cython  # NOQA
 from libcpp cimport vector
 
 from cupy_backends.cuda.api cimport driver
+from cupy_backends.cuda.api cimport runtime
```

Comment:
I think this import is no longer needed?

Reply:
Good catch, I removed those now.